src/components/navigation/JumpX.ts

```diff
@@ -16,7 +16,7 @@
  * @param {string[]} series
  * @return {void}
  */
-export function jumpXpoints(
+export function jumpXelements(
```

Comment:
Naming here is also not very consistent.
Shouldn't it be `jumpXElements`? (with capital E)
Or use a longer name like `jumpForXElements`, for instance